bipen/src/main/java/com/lftechnology/phpjava/assignmenttwo/SalutationSplitter.java

```diff
@@ -0,0 +1,49 @@
+package com.lftechnology.phpjava.assignmenttwo;
+
+import java.util.Scanner;
+
+/**
+ * Split salutation given full name
+ * 
+ * @author bipen
+ *
+ */
+public class SalutationSplitter {
+    protected String salutation;
+    protected String inputName;
+    protected String fullName;
+
+    /**
+     * Main Method to split salutation from full name and displays result
+     */
+    public void run() {
+        System.out.println("Please enter Full Name with salutation(Mr Bipen Chhetri):");
+        Scanner name = new Scanner(System.in);
+        this.inputName = name.nextLine();
+        this.validateInput();
+        System.out.println("Your Salutation is: " + this.salutation);
+        System.out.println("Your Full Name is: " + this.fullName);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)